var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -51,6 +51,7 @@ class Paraview(Package):
     depends_on('qt@:4', when='+qt')
 
     depends_on('cmake', type='build')
+    depends_on('mesa')
```

Comment:
We might need a virtual dependency for `opengl`, and a set of recommendations for setting this stuff up.  This was an indirect dependency already via `qt`; curious why @certik had to add it to `paraview`.